From f366eed4db053c811d43f1deb21c30dae66c9e1a Mon Sep 17 00:00:00 2001
From: "Johan T. Halseth" <johanth@gmail.com>
Date: Fri, 6 Apr 2018 16:26:29 +0200
Subject: [PATCH] enable mainnet neutrino

---
 config.go | 7 -------
 1 file changed, 7 deletions(-)

diff --git a/config.go b/config.go
index a65123be84..bfc58615c6 100644
--- a/config.go
+++ b/config.go
@@ -808,13 +808,6 @@ func loadConfig() (*config, error) {
 			return nil, err
 		}
 
-		if cfg.Bitcoin.Node == "neutrino" && cfg.Bitcoin.MainNet {
-			str := "%s: neutrino isn't yet supported for " +
-				"bitcoin's mainnet"
-			err := fmt.Errorf(str, funcName)
-			return nil, err
-		}
-
 		if cfg.Bitcoin.TimeLockDelta < minTimeLockDelta {
 			return nil, fmt.Errorf("timelockdelta must be at least %v",
 				minTimeLockDelta)
